internal/pkg/agent/cmd/enroll.go

```diff
@@ -299,12 +323,14 @@ func enroll(streams *cli.IOStreams, cmd *cobra.Command) error {
 	fElasticSearchInsecure, _ := cmd.Flags().GetBool("fleet-server-es-insecure")
 	fHeaders, _ := cmd.Flags().GetStringSlice("header")
 	fServiceToken, _ := cmd.Flags().GetString("fleet-server-service-token")
+	fServiceTokenFile, _ := cmd.Flags().GetString("fleet-server-service-token-file")
 	fPolicy, _ := cmd.Flags().GetString("fleet-server-policy")
 	fHost, _ := cmd.Flags().GetString("fleet-server-host")
 	fPort, _ := cmd.Flags().GetUint16("fleet-server-port")
 	fInternalPort, _ := cmd.Flags().GetUint16("fleet-server-internal-port")
 	fCert, _ := cmd.Flags().GetString("fleet-server-cert")
 	fCertKey, _ := cmd.Flags().GetString("fleet-server-cert-key")
+	fPassphrase, _ := cmd.Flags().GetString("fleet-server-cert-key-passphrase-file")
```

Comment:
Being that we don't offer a way of providing a passphrase without it being in a file. I don't know if this needs the `-file` suffix. The `--fleet-server-cert` does not end in `-file` but it is always a path to a file.

Reply:
Set to requested changes, only for this one. I don't believe it should have `-file` at the end.